src/AccessibilityInsights.SharedUx/ViewModels/BaseActionViewModel.cs

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using AccessibilityInsights.CommonUxComponents.Dialogs;
-using AccessibilityInsights.SharedUx.Dialogs;
```

Comment:
These using blocks were already unneeded--I just cleaned them up while I was in the file